AdminUi/test/AdminUi.Tests.Integration/StepDefinitions/ClientDetailsStepDefinitions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using AdminUi.Tests.Integration.API;
+using AdminUi.Tests.Integration.Extensions;
+using AdminUi.Tests.Integration.Models;
+
+namespace AdminUi.Tests.Integration.StepDefinitions;
+public class ClientDetailsStepDefinitions : BaseStepDefinitions
+{
+    private readonly ClientsApi _clientsApi;
+    private string _clientId;
+    private HttpResponse<ClientDTO>? _response;
+
+    public ClientDetailsStepDefinitions(ClientsApi clientsApi)
+    {
+        _clientsApi = clientsApi;
+        _clientId = string.Empty;
+    }
+
+    [Given(@"a Client c")]
+    public async Task GivenAClient()
+    {
+        var createClientRequest = new CreateClientRequest
+        {
+            ClientId = string.Empty,
+            DisplayName = "a-client-display-name",
```

Comment:
You should pass as little data as necessary, in order to reduce noise in the tests. The display name isn't required, so you shouldn't pass it.